staging/src/github.com/kubeedge/beehive/pkg/core/model/message.go

```diff
@@ -253,13 +253,20 @@ func (msg *Message) GetDestination() string {
 }
 
 // String the content that you want to send
-func (msg *Message) String() string {
+func (msg Message) String() string {
```

Comment:
Remove the pointer is not reasonable :) We should use the pointer.

Reply:
There are two reasons why I prefer the value receiver:

1. You can invoke the `Stirng` method on both `msg` and `&msg`. If you define the method on pointer, you can only invoke it on `&msg`.
2. The value receiver indicates that `msg` is immutable, i.e. we won't change the `msg` passed by the caller.